pinot-core/src/main/java/com/linkedin/pinot/core/common/MinionConstants.java

```diff
@@ -23,6 +23,7 @@ private MinionConstants() {
   public static final String SEGMENT_NAME_KEY = "segmentName";
   public static final String DOWNLOAD_URL_KEY = "downloadURL";
   public static final String UPLOAD_URL_KEY = "uploadURL";
+  public static final String CRC_KEY = "crc";
```

Comment:
please add some comments on CRC_KEY. Also, suggest renaming it to ORIGINAL_SEGMENT_CRC_KEY